src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/Utgifter/UtgifterValg.tsx

```diff
@@ -55,53 +63,24 @@ const UtgifterValg: React.FC<Props> = ({ utgifter, barn, errorState, oppdaterUtg
         });
     };
 
-    // Oppdater for riktig rad
-    // Returner hele utgift objekt?
-
-    // const leggTilTomRadUnder = () => {
-    //     // const
-    //     if (utgifter) oppdaterUtgifter([...utgifter, { fra: '', til: '' }]);
-    //     else oppdaterUtgifter([{ fra: '', til: '' }]);
-    // };
-
-    // const periodeVariantTilUtgiftsperiodeProperty = (
-    //     periodeVariant: PeriodeVariant
-    // ): EUtgiftsperiodeProperty => {
-    //     switch (periodeVariant) {
-    //         case PeriodeVariant.ÅR_MÅNED_FRA:
-    //             return EUtgiftsperiodeProperty.årMånedFra;
-    //         case PeriodeVariant.ÅR_MÅNED_TIL:
-    //             return EUtgiftsperiodeProperty.årMånedTil;
-    //     }
-    // };
-
-    // const lukkSanksjonsmodal = () => {
-    //     settSanksjonsmodal({ visModal: false });
-    // };
-
-    // const slettPeriode = (indeks: number) => {
-    //     if (sanksjonsmodal.visModal) {
-    //         lukkSanksjonsmodal();
-    //     }
-    //     utgiftsperioderState.remove(indeks);
-    //     settValideringsFeil((prevState: FormErrors<InnvilgeVedtakForm>) => {
-    //         const utgiftsperioder = (prevState.utgiftsperioder ?? []).filter((_, i) => i !== indeks);
-    //         return { ...prevState, utgiftsperioder };
-    //     });
-    // };
-
-    // const slettPeriodeModalHvisSanksjon = (indeks: number) => {
-    //     const periode = utgiftsperioderState.value[indeks];
-    //     if (periode.periodetype === EUtgiftsperiodetype.SANKSJON_1_MND) {
-    //         settSanksjonsmodal({
-    //             visModal: true,
-    //             indeks: indeks,
-    //             årMånedFra: periode.årMånedFra,
-    //         });
-    //     } else {
-    //         slettPeriode(indeks);
-    //     }
-    // };
+    const leggTilTomRadUnder = (utgiftIndex: number) => {
+        oppdaterUtgiter(leggTilTomRadUnderIListe(utgifter, tomUtgiftRad(), utgiftIndex));
```

Comment:
Usikker på om det burde stå `prevState = utgifter` over mtp mutering? I stedet for å sende inn utgifter direkte her